src/navigation/index.js

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { NavigationContainer, DefaultTheme } from '@react-navigation/native';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import Theme from '../theme/index';
+// import FontAwesome5 from 'react-native-vector-icons/FontAwesome5 ';
+// import Ionicons from 'react-native-vector-icons/Ionicons';
```

Comment:
If I remember correctly, React Navigation does come with build in icons: https://reactnavigation.org/docs/tab-based-navigation#customizing-the-appearance. Which may be a stop-gap until we can solve the icon strategy in a holistic way.